hw/ip/rv_dm/doc/checklist.md

```diff
@@ -0,0 +1,235 @@
+---
+title: "RV_DM Checklist"
+---
+
+This checklist is for [Hardware Stage]({{< relref "/doc/project/development_stages.md" >}}) transitions for the [RV_DM peripheral.]({{< relref "." >}})
+All checklist items refer to the content in the [Checklist.]({{< relref "/doc/project/checklist.md" >}})
+
+## Design Checklist
+
+### D1
+
+Type          | Item                           | Resolution  | Note/Collaterals
+--------------|--------------------------------|-------------|------------------
+Documentation | [SPEC_COMPLETE][]              | Done        | [RV_DM Design Spec]({{<relref "." >}})
+Documentation | [CSR_DEFINED][]                | Done        |
+RTL           | [CLKRST_CONNECTED][]           | Done        |
+RTL           | [IP_TOP][]                     | Done        |
+RTL           | [IP_INSTANTIABLE][]            | Done        |
+RTL           | [PHYSICAL_MACROS_DEFINED_80][] | Done        |
```

Comment:
Does the debug module have any macros? I think the debug ROM itself is implemented using a sea of gates. Thus, I would put a `N/A` here and under `Note/Collaterals` say something like "Debug ROM implemented using sea of gates"